pretyping/detyping.ml

```diff
@@ -643,6 +642,19 @@ let it_destRLambda_or_LetIn_names l c =
               | _ -> DAst.make @@ GApp (c,[a]))
   in aux l [] c
 
+let safe_lookup_mind_specif env ind =
+  if Environ.mem_mind (fst ind) env then Inductive.lookup_mind_specif env ind
+  else raise Exit
```

Comment:
Doesn't this behave pretty badly in the debugger? It seems like any time we try to print a term that has a case node we will get a printer exception.

eg
~~~
Eval cbv in nat_rect.
~~~
breakpoint at https://github.com/coq/coq/blob/24b7eac498ec46b69f3d411c2aeb6a36122248f4/vernac/vernacentries.ml#L1911 and print `c` -> `<printer Top_printers.ppeconstr raised an exception: Stdlib.Exit>`